src/hooks/useScroll.js

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from 'react';
+
+const useScroll = () => {
+  const [isNavOn, setIsNavOn] = useState(false);
+
+  const handleScroll = () => {
+    if (window.scrollY > 380) {
+      setIsNavOn(true);
+      return;
+    }
+    setIsNavOn(false);
+    return;
+  };
+
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll);
+    return () => {
+      window.removeEventListener('scroll', handleScroll);
+    };
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    const handleScroll = () => {
      if (window.scrollY > 380) {
        setIsNavOn(true);
        return;
      }
      setIsNavOn(false);
    };
  
    window.addEventListener('scroll', handleScroll);
    return () => {
      window.removeEventListener('scroll', handleScroll);
    };
  }, []);
```
useEffect 안쪽으로 함수 옮기면 의존성 배열을 더 정확하게 써줄 수 있을거 같아요